client/src/store/providers/apollo.tsx

```diff
@@ -7,14 +7,16 @@ import {
   NormalizedCacheObject,
 } from '@apollo/client';
 import { setContext } from '@apollo/client/link/context';
+import get from 'lodash/get'
 import { RptManager } from '../../helpers/keycloak-api/manager';
 import { GraphqlBackend, GraphqlProvider } from 'store/providers';
 
-const ARRANGER_API = process.env.REACT_APP_ARRANGER_API;
-const PROJECT_ID = process.env.REACT_APP_ARRANGER_PROJECT_ID;
+const ARRANGER_API = get(window, 'CLIN.arrangerBaseUrl', process.env.REACT_APP_ARRANGER_API)
```

Comment:
I think it would be clearer thant using object access via a string path: window?.CLIN?.arrangerBaseUrl || process.env.REACT_APP_ARRANGER_API